### DISCUSSION TITLE
BTT smart filament sensor with manual mesh

### DISCUSSION THREAD

**Discussion by Bezur-Ro** [Jyers/Marlin#1261] [Q&A] [answered]
I need help compiling the manual mesh 5x5 FW so that it works with the smart filament sensor

**Comment by Cal-LF** [comment, marked as answer]
@Bezur-Ro If you want to try mine, I have the manual mesh 4X4 with the BTT sensor. Change the .zip to .bin
[firmware-20210730 4X4 Man.zip](https://github.com/Jyers/Marlin/files/6913570/firmware-20210730.4X4.Man.zip)
If this does not work, I can try to compile one with slightly different settings for you.
